teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Person initializer accepts `age, name, parent_persmission` as parameters but you are passing `name, age, parent_permission` in the super method, It will swap the age with the name. Please fix this issue and pass the parameters in the super method in this sequence `super(age, name, parent_permission)`

Reply:
@thecodechaser thank you, I have fixed it.